### DISCUSSION TITLE
Triggers and functions

### DISCUSSION THREAD

**Discussion by gowizzard** [redis/go-redis#2712] [Ideas]
Hello all, I haven't found anything about this here yet, but with the new 7.2 version and the triggers and features that are now available with Redis Gears, I wonder if this will be added? From Redis 8.0 this should also be available for production.

Thanks a lot
Jonas

**Comment by ofekshenawa** [comment]
Hi @gowizzard,

I wanted to let you know that we added Triggers and Functions support in v9.1.0. Additionally, we've just released go-redis version 9.2, which includes Timeseries and Probabilistic modules and also support for some more Redis 7.0 commands.

You can find more details in the release notes here:

https://github.com/redis/go-redis/releases/tag/v9.2.0
